### PR TITLE
EPMLSTRTAI-864: added changeSets for creating default roles and default admin

### PR DESCRIPTION
**Pull request by PharaonSP**
EPMLSTRTAI-864: added changeSets for creating default roles and default admin
deleted InitDataLoader.java

### CODE REVIEW

File: bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml [hunk 1]
```diff
@@ -214,4 +214,32 @@
             </column>
         </createTable>
     </changeSet>
+
+    <changeSet author="" id="create_default_roles">
```
Comment (hdghg) [inline bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml]: Please fill value of author tag

File: bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml [hunk 2]
```diff
@@ -214,4 +214,32 @@
...
+        </insert>
+        <insert tableName="ROLE">
+            <column name="NAME" value="GUEST"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="" id="create_default_admin">
+        <insert tableName="USER">
+            <column name="EMAIL" value="admin@mail.com"/>
+            <column name="PASSWORD" value="$2a$10$b8myM0VkLZTkSAgQ2Ijs.e9t.WGvXVnPKtJDQMZ7iJWdZ6uVZRX6u"/>
```
Comment (hdghg) [inline bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml]: Please document default admin login and password in project files, i.e. README.md

File: bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml (hunk 1 above)
Comment (PharaonSP) [reply bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml]: done

File: bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml (hunk 2 above)
Comment (PharaonSP) [reply bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml]: done

**Pull request merged**